src/patterns/Subscribe/Subscribe.tsx

```diff
@@ -72,6 +72,7 @@ export interface SubscribeProps extends React.HTMLAttributes<HTMLFormElement> {
  */
 
 const Subscribe = ({
+  autoFocus = true,
```

Comment:
would this ever get consumed somewhere like the footer?

might be weird to autofocus an element that's off screen unintentionally, if this is defaulting to true

Reply:
We're only planning to show it in some kind of Drawer at this point so I think this is a good default.